rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -19,9 +19,68 @@ add a new schema rather than change existing material.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Which fieldsets will be impacted? How many fields overall? Are we primarily adding fields, removing fields, or changing existing fields? The goal here is to understand the fundamental technical implications and likely extent of these changes. ~2-5 sentences.
--->
+The proposed change adds six fields, as described below:
+
+```
+---
+- name: orchestrator
+  title: Orchestrator
+  group: 2
+  short: Fields relevant to container orchestrators.
+  description: >
+    Fields that describe the resources which container orchestrators manage or
+    act upon.
+  type: group
+  fields:
+    - name: cluster
+      level: core
+      type: keyword
+      example: production
+      description: >
+        Orchestrator cluster name.
```

Comment:
Make this an object in case it includes more data about the cluster in the future? e.g. it could include the name, some other identifier, the endpoint, the version...
See https://github.com/elastic/beats/issues/17467#issuecomment-610381848

Reply:
Agreed! Maybe for your `example: production` here, it can be a more specific field under `cluster` object. For example: `orchestrator.cluster.environment` in this case.

Reply:
Sounds good to me! Will add this shortly.

Reply:
I have made the update, but a question occurred to me as I was doing so - would it make sense to define the common expected fields that would come under the `cluster` object, or is best practice to keep it as open as possible for flexibility? I've kept it open for now.

Reply:
Yes please, add the common expected fields under the `cluster` object. At least `name` and `url` seem to be relevant.